config/static/nodes/rpm-overrides/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Generate the head of this file into the Makefile, remove this file from the repository, and add this file to .gitignore file.